data_steward/analytics/cdr_ops/ehr_snapshot_qc.py

```diff
@@ -89,3 +89,42 @@
 ORDER BY table_id
 '''
 execute(client, query)
+
+# # Person_to_observation records observation_date and observation_datetime check
+# Check to make sure records added to observation with observation_concept_id in (4013886, 421761, 4135376, 4083587) are
+# not using participant’s birth date, as seen in person.birth_datetime.
+# Check is successful when result is empty
+
+query = f"""
+WITH
+  person_birth_date AS (
+  SELECT
+    birth_datetime,
+    DATE(birth_datetime) AS birth_date
+  FROM
+    `{PROJECT_ID}.{EHR_SNAPSHOT_DATASET_ID}.unioned_ehr_person` )
+SELECT
+  observation_id,
+  person_id,
+  observation_concept_id,
+  observation_date,
+  observation_datetime
+FROM
+  `{PROJECT_ID}.{EHR_SNAPSHOT_DATASET_ID}.unioned_ehr_observation`
+WHERE
+  observation_concept_id IN (4013886,
+    421761,
```

Comment:
Though the ticket says it is `421761`, I think this needs to be `4271761` as Jason's change for line 576:
https://github.com/all-of-us/curation/pull/1117/files#diff-33687d7723451a4637751a9d35dc8f85aa9ce939865e4cb0770f9b454220ed33L576
Could you please double-check if this is the correct ID?